tests/defaults.py

```diff
@@ -1,93 +0,0 @@
-# Copyright 2018 Xanadu Quantum Technologies Inc.
-
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-
-#     http://www.apache.org/licenses/LICENSE-2.0
-
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-"""
-Default parameters, commandline arguments and common routines for the unit tests.
-"""
-
-import argparse
-import unittest
-import os
-import sys
-import logging
-
-import numpy as np
-
-# Make sure pennylane is always imported from the same source distribution where the tests reside, not e.g. from site-packages.
-# See https://docs.python-guide.org/en/latest/writing/structure/#test-suite
-sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
-
-import pennylane_sf
-
-
-# defaults
-TOLERANCE = os.environ.get("TOL", 1e-5)
-
-
-# set up logging
-if "LOGGING" in os.environ:
-    logLevel = os.environ["LOGGING"]
-    print('Logging:', logLevel)
-    numeric_level = getattr(logging, logLevel.upper(), 10)
-else:
-    numeric_level = 100
-
-logging.getLogger().setLevel(numeric_level)
-logging.captureWarnings(True)
```

Comment:
thought that logging would be taken care of by `pytest`

Reply:
Yep, fantastic!